src/pages/workspaces/workspace/Tools.js

```diff
@@ -85,6 +85,12 @@ const ToolCard = pure(({ listView, name, namespace, config, onCopy, onDelete })
       })
     ])
   ])
+
+  const repoLink = link({
+    href: sourceRepo === 'agora' ? `https://firecloud.dsde-dev.broadinstitute.org/#methods/${methodNamespace}/${methodName}/${methodVersion}`: `https://dockstore.org/workflows/${methodPath}`,
```

Comment:
This should get the firecloud and dockstore roots from the config so it uses the right urls for each environment. Also, we should pull this logic out into a utility function so we can use it in both places.